package.json

```diff
@@ -1,25 +1,31 @@
 {
   "name": "route-recognizer",
-  "version": "0.2.8",
+  "version": "0.3.0",
```

Comment:
I did that just because I changed the dist not to bother with named-amd (I added that because I thought I was going to use it and not do the es6 dir, vs just a file (souremap was easier).